platform-dom/src/main/java/com/softicar/platform/dom/elements/image/viewer/DomImageViewerPageNavigationBar.java

```diff
@@ -0,0 +1,41 @@
+package com.softicar.platform.dom.elements.image.viewer;
+
+import com.softicar.platform.dom.DomCssClasses;
+import com.softicar.platform.dom.DomImages;
+import com.softicar.platform.dom.DomTestMarker;
+import com.softicar.platform.dom.elements.bar.DomBar;
+import com.softicar.platform.dom.elements.button.DomButton;
+
+class DomImageViewerPageNavigationBar extends DomBar {
+
+	private final DomImageViewer viewer;
+	private final DomButton nextImageButton;
+	private final DomButton previousImageButton;
+	private final DomImageViewerPageIndicator pageIndicator;
+
+	public DomImageViewerPageNavigationBar(DomImageViewer viewer) {
+
+		this.viewer = viewer;
+		this.previousImageButton = new DomButton()//
+			.addMarker(DomTestMarker.IMAGE_VIEWER_PREVIOUS_PAGE_BUTTON)
+			.setIcon(DomImages.PAGE_PREVIOUS.getResource())
+			.setClickCallback(() -> viewer.showPreviousPage());
+		this.nextImageButton = new DomButton()//
+			.addMarker(DomTestMarker.IMAGE_VIEWER_NEXT_PAGE_BUTTON)
+			.setIcon(DomImages.PAGE_NEXT.getResource())
+			.setClickCallback(() -> viewer.showNextPage());
+		this.pageIndicator = new DomImageViewerPageIndicator(viewer);
+
+		addCssClass(DomCssClasses.DOM_IMAGE_VIEWER_NAVIGATION_BAR);
+		appendChild(previousImageButton);
+		appendChild(pageIndicator);
```

Comment:
I think the element order should be:
`[prev-button] [next-button] [page-indicator]`
That way, paging forth and back is easier (because no need to let the cursor travel across an output element).
....much like in the zoom control element. And much like in pdf.js:
![image](https://user-images.githubusercontent.com/15086658/188463994-9d506cb3-62ef-405e-bb6a-0e1e79c0a78e.png)

Reply:
as you wish :)